docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -14,10 +15,10 @@ box Model MODEL_COLOR_T1
 participant "m:Model" as Model MODEL_COLOR
 end box
 
-[-> LogicManager : execute("delete 1")
+[-> LogicManager : execute("delete E01234567")
```

Comment:
Remember to update this to `delete id/E01234567` to reflect the change in the `DeleteCommand` :)